Tweak the @parent variable description (#3753)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/expression.rst

 map_units                    Returns the units of map measurements
 notification_message         Content of the notification message sent by the provider
                              (available only for actions triggered by provider notifications).
 parent                       Refers to the current feature in the parent layer,

>                               allowing to fetch its attribute or geometry when filtering

Reviewer: ```suggestion
                              providing access to its attributes and geometry when filtering
```
